app/src/controllers/launch/actionCreators.js

```diff
@@ -27,6 +27,16 @@ export const proceedWithValidItemsAction = createProceedWithValidItemsAction(NAM
 
 const MODAL_COMPARE_WIDTH = 900;
 
+export const finishForceLaunchesAction = defineGroupOperation(
```

Comment:
forceFinishLaunchesAction? I think it sounds better

Reply:
Fixed